spec/models/customer_spec.rb

```diff
@@ -616,12 +616,14 @@
 
     let(:organization) { create(:organization, timezone: "America/Los_Angeles") }
```

Comment:
we can remove the timezone for the organization to avoid confusion when reading the specs.

```suggestion
    let(:organization) { create(:organization) } 
```